indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -75,10 +76,19 @@ protected void map(
         if (reportParseExceptions) {
           throw e;
         }
-        log.debug(e, "Ignoring invalid row [%s] due to parsing error", value.toString());
+        if (log.isDebugEnabled()) {
```

Comment:
log.isDebugEnabled() check is built-in inside log.debug(..) so this is not necessary. you probably wanted to avoid eager `value.toString()` computation that can be changed by replacing same with `value` .

Reply:
Yes, this was the reason. Good point, I'll change it to just `value`.